spec/Interoperation.tex

```diff
@@ -382,8 +382,8 @@ \subsection{Shared Data}
 constants.
 
 \begin{craychapel}
-Note that since params must be known to Chapel at compile-time (and
-because the Chapel compiler doesn't have the ability to parse C code),
+Note that since params must be known to Chapel at compile-time and
+the Chapel compiler does not parse external C code,
```

Comment:
I think this is a funny comment because with the LLVM infrastructure, the Chapel compiler *does* parse external C code. It's a nit, but it could be updated to say "the Chapel compiler does not necessarily parse external C code".